astropy/tests/runner.py

```diff
@@ -377,7 +393,7 @@ def pastebin(self, pastebin, kwargs):
     def remote_data(self, remote_data, kwargs):
         """
         remote_data : {'none', 'astropy', 'any'}, optional
-            Controls whether to run tests marked with @remote_data. This can be
+            Controls whether to run tests marked with @pytest.mark.remote_data. This can be
```

Comment:
so, now both ``@remote_data`` and `pytest.mark.remote_data` would work? Since we don't yet rely on `pytest-astropy` here, probably better to leave this as is?